src/containers/Application/ApplicationHome.tsx

```diff
@@ -45,25 +49,32 @@ const ApplicationHome: React.FC<ApplicationProps> = ({ structure, template }) =>
 
   if (!fullStructure || !fullStructure.responsesByCode) return <Loading />
 
-  const canUserEdit = fullStructure.info?.current?.status === ApplicationStatus.Draft
-
-  const { firstStrictInvalidPage } = fullStructure.info
+  const {
+    info: { current, isChangeRequest, firstStrictInvalidPage },
+    sections,
+  } = fullStructure
 
   const HomeMain: React.FC = () => {
     return (
       <>
+        <ChangesRequestedTitle status={current?.status} isChangeRequest={isChangeRequest} />
         <Segment>
           <Header as="h5">{strings.SUBTITLE_APPLICATION_STEPS}</Header>
           <Header as="h5">{strings.TITLE_STEPS.toUpperCase()}</Header>
           <SectionsProgress
-            canEdit={canUserEdit}
-            sections={fullStructure.sections}
+            changesRequested={isChangeRequest}
+            draftStatus={current?.status === ApplicationStatus.Draft}
+            sections={sections}
             firstStrictInvalidPage={firstStrictInvalidPage}
+            restartApplication={async ({ sectionCode, pageNumber }) => {
+              await restartApplication(fullStructure)
```

Comment:
would be good to do some error checking (as per example in the first comment in this PR)